golem-cli/src/lib.rs

```diff
@@ -55,15 +55,27 @@ pub fn version() -> &'static str {
     }
 }
 
-pub fn init_tracing(verbosity: Verbosity) {
+pub fn init_tracing(verbosity: Verbosity, pretty_mode: bool) {
```

Comment:
i'm fine with these defaults for your preference for now, but later i think we should add the golem-common env var based tracing config, so anybody can tweak to their liking (which if for me probably will be json), and because i think the pretty mode is overly "chatty" for most

Reply:
(also, happy to do this later)

Reply:
It's not showing anything more than the "non pretty" version just in multiple lines so it is actually readable. In the previous default mode I find it impossible to find the actual log messages (without a search tool).
The verbosity parameter still works independent of this.